packages/core/src/cookie.ts

```diff
@@ -77,15 +79,21 @@ export function areCookiesAuthorized(options: CookieOptions): boolean {
  * strategy: find the minimal domain on which cookies are allowed to be set
  * https://web.dev/same-site-same-origin/#site
  */
+let getCurrentSiteCache: string | undefined
 export function getCurrentSite() {
-  const testCookieName = 'dd_site_test'
-  const testCookieValue = 'test'
+  if (getCurrentSiteCache === undefined) {
```

Comment:
what about turning it into an early return?

Reply:
not really important, not changing the indentation would have make your commit easier to review

Reply:
Hiding whitespace changes helps in this case https://github.com/DataDog/browser-sdk/pull/573/files?w=1